lib/kafka/async_producer.rb

```diff
@@ -218,6 +218,12 @@ def run
             raise "Unknown operation #{operation.inspect}"
           end
         end
+      rescue Kafka::Error => e
+        @logger.error "Unexpected Kafka error #{e.class}: #{e.message}\n#{e.backtrace.join("\n")}"
```

Comment:
👍 to logging this error. Doesn't have to be now, but we should publish these errors out through the instrumenter.

Reply:
Yeah, it would make sense to track this in a dashboard.